src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -194,7 +194,8 @@
 
         az image builder create --image-source $imagesource -n mytemplate -g my-group \\
             --scripts $scripts --managed-image-destinations image_1=westus \\
-            --shared-image-destinations my_shared_gallery/linux_image_def=westus,brazilsouth
+            --shared-image-destinations my_shared_gallery/linux_image_def=westus,brazilsouth \\
+            --identity myidentity
```

Comment:
Is it possible to make `--identity` a required field for `az image builder create` cmd?

Reply:
No problem

Reply:
Done

Reply:
I find a problem. When using template to pass parameters, --identity is not required.

Reply:
makes sense.